WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/ProductDetailsTableViewCell.swift

```diff
@@ -45,6 +45,12 @@ final class ProductDetailsTableViewCell: UITableViewCell {
         configureSubtitleLabel()
         configureSelectionStyle()
     }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        findSectionTopSeparator()?.isHidden = true
```

Comment:
Thank you, @jaclync! 🙏 That is so much better because the hack is encapsulated within `PrimarySectionHeaderView`. 🙇 

I added that in 79f82daad60df82ce01e92e9c22c789f46020982. 

This caused a lot of code to be removed. Since I had some more space, I played around and some Xcode previews. 😁 

https://github.com/woocommerce/woocommerce-ios/blob/2731596ac3c065bc0bd0dd9adbb86d90a8100169/WooCommerce/Classes/ViewRelated/ReusableViews/Section%20Headers/PrimarySectionHeaderView.swift#L56-L103